dev-workflow-ui-web-test/src_test/ch/ivyteam/ivy/project/workflow/webtest/WebTestTasksIT.java

```diff
@@ -241,6 +241,14 @@ void customFieldEmbedInFrame() {
 
     Navigation.openTask("notEmbedTask");
     $(By.id("actionMenuForm:taskStartBtn")).shouldNotHave(text("?taskUrl"));
+
+    var taskStartLink = $(By.id("actionMenuForm:taskStartBtn")).find(By.tagName("a")).getAttribute("href");
+    Selenide.open(taskStartLink);
+    $(By.id("iFrameForm")).shouldNotBe(visible);
+
+    taskStartLink = taskStartLink + "&embedInFrame";
+    Selenide.open(taskStartLink);
+    $(By.id("iFrameForm")).shouldNotBe(visible);
```

Comment:
this should only be the case when the customField `embedInFrame=false` is set on the task or case!